<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindow</class>
 <widget class="QMainWindow" name="SettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>828</width>
    <height>823</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>796</width>
         <height>805</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>12</number>
        </property>
        <property name="topMargin">
         <number>6</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <pointsize>18</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Javascript</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="topMargin">
           <number>6</number>
          </property>
          <property name="bottomMargin">
           <number>12</number>
          </property>
          <item>
           <widget class="QRadioButton" name="radio_js_enabled">
            <property name="text">
             <string>Enabled</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup_3</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radio_js_disabled">
            <property name="text">
             <string>Disabled</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup_3</string>
            </attribute>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="font">
           <font>
            <pointsize>18</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Adblock: uBlockOrigin</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="check_adblock_filters">
          <property name="text">
           <string>Filters</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="check_adblock_abp">
          <property name="text">
           <string>I don't care about cookies</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="check_adblock_privacy">
          <property name="text">
           <string>Privacy filters</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="check_adblock_abuse">
          <property name="text">
           <string>Resource abuse</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="check_adblock_unbreak">
          <property name="text">
           <string>Unbreak</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>18</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>User Agent</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="topMargin">
           <number>6</number>
          </property>
          <property name="bottomMargin">
           <number>12</number>
          </property>
          <item>
           <widget class="QRadioButton" name="radio_ua_desktop">
            <property name="text">
             <string>Desktop</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radio_ua_mobile">
            <property name="text">
             <string>Mobile</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>18</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Zoom</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="topMargin">
           <number>6</number>
          </property>
          <property name="bottomMargin">
           <number>12</number>
          </property>
          <item>
           <widget class="QRadioButton" name="radio_zoom_75">
            <property name="text">
             <string>75%</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup_2</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radio_zoom_100">
            <property name="text">
             <string>100%</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup_2</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radio_zoom_125">
            <property name="text">
             <string>125%</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup_2</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radio_zoom_150">
            <property name="text">
             <string>150%</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup_2</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radio_zoom_200">
            <property name="text">
             <string>200%</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup_2</string>
            </attribute>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <pointsize>18</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Allow running insecure content</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="hor5">
          <property name="topMargin">
           <number>6</number>
          </property>
          <property name="bottomMargin">
           <number>12</number>
          </property>
          <item>
           <widget class="QRadioButton" name="radio_insecurecontent_enabled">
            <property name="text">
             <string>Enabled</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup_4</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radio_insecurecontent_disabled">
            <property name="text">
             <string>Disabled</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup_4</string>
            </attribute>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <pointsize>18</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>PDF viewer</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="hor6">
          <property name="topMargin">
           <number>6</number>
          </property>
          <property name="bottomMargin">
           <number>12</number>
          </property>
          <item>
           <widget class="QRadioButton" name="radio_pdfviewer_enabled">
            <property name="text">
             <string>Enabled</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup_5</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radio_pdfviewer_disabled">
            <property name="text">
             <string>Disabled</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup_5</string>
            </attribute>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <pointsize>18</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Scrollbar</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="hor7">
          <property name="topMargin">
           <number>6</number>
          </property>
          <property name="bottomMargin">
           <number>12</number>
          </property>
          <item>
           <widget class="QRadioButton" name="radio_scrollbar_enabled">
            <property name="text">
             <string>Enabled</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup_6</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radio_scrollbar_disabled">
            <property name="text">
             <string>Disabled</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup_6</string>
            </attribute>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <pointsize>18</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>WebGL</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="topMargin">
           <number>6</number>
          </property>
          <property name="bottomMargin">
           <number>12</number>
          </property>
          <item>
           <widget class="QRadioButton" name="radio_webgl_enabled">
            <property name="text">
             <string>Enabled</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup_7</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radio_webgl_disabled">
            <property name="text">
             <string>Disabled</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup_7</string>
            </attribute>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>linkClicked()</slot>
 </slots>
 <buttongroups>
  <buttongroup name="buttonGroup_7"/>
  <buttongroup name="buttonGroup_3"/>
  <buttongroup name="buttonGroup"/>
  <buttongroup name="buttonGroup_4"/>
  <buttongroup name="buttonGroup_6"/>
  <buttongroup name="buttonGroup_2"/>
  <buttongroup name="buttonGroup_5"/>
 </buttongroups>
</ui>
